apache/trafficcontrol
# Add `.py` file extension to Python Postinstall script

Closes: #6102

This PR
* adds a `.py` file extension to the Python Postinstall script,
* Uses the Python Postinstall script in the *Traffic Portal Integration Tests* GitHub Action, and
* Removes the [`adduser.pl`](https://github.com/apache/trafficcontrol/blob/871659244b/infrastructure/cdn-in-a-box/traffic_ops/adduser.pl) from CDN in a Box. It is no longer needed since #5681, which initializes Traffic Ops in CDN in a Box using the Traffic Ops Postinstall script.
-----------------------------------------------


## Which Traffic Control components are affected by this PR?

- Traffic Ops
- CDN in a Box
- Automation (GitHub Actions)

## What is the best way to verify this PR?

* Verify that text editors recognize the Postinstall script as being a Python script:
    ```shell
    vim -c 'echo &syntax' traffic_ops/install/bin/_postinstall.py
    ```

* Verify that CDN in a Box still works, now that `adduser.pl` is removed

* Verify that the *TP Integration Tests* GHA succeeds now that the TO admin username and password are gotten from the TP integration tests' `config.json` config file

## PR submission checklist
- [x] This PR modifies tests
- [x] Usage unchanged and `_postinstall` was not mentioned in the documentation, no documentation is needed needed 
- [x] No changelog entry needed
